talkvoter/settings.py

```diff
@@ -1,6 +1,9 @@
 import os
 
+SECRET_KEY = os.getenv("SECRET_KEY")
 
 SQLALCHEMY_DATABASE_URI = os.getenv("DATABASE_URL", "sqlite:///:memory:")
 
 SQLALCHEMY_TRACK_MODIFICATIONS = False
+
+ERROR_404_HELP = False
```

Comment:
Flask has a few config idioms.

https://github.com/alysivji/flask-mega-tutorial/blob/master/app/config.py to create
https://github.com/alysivji/flask-mega-tutorial/blob/master/app/__init__.py to use

Reply:
updated config